common/openapi.yml

```diff
@@ -178,13 +178,13 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/AsyncPipelineRunResponse'
+                $ref: '#/components/schemas/AsyncPipelineRunResponseV2'
```

Comment:
if we're changing the response of this endpoint doesnt that mean we need to update the endpoint to v2?